labgrid/driver/power/sentry.py

```diff
@@ -0,0 +1,94 @@
+from ..exception import ExecutionError
+from ...util.helper import processwrapper
+
+INDEX_TO_OID = {
+    1: "1.1.1",
+    2: "1.1.2",
+    3: "1.1.3",
+    4: "1.1.4",
+    5: "1.1.5",
+    6: "1.1.6",
+    7: "1.1.7",
+    8: "1.1.8",
+    9: "1.2.1",
+    10: "1.2.2",
+    11: "1.2.3",
+    12: "1.2.4",
+    13: "1.2.5",
+    14: "1.2.6",
+    15: "1.2.7",
+    16: "1.2.8",
+    17: "1.3.1",
+    18: "1.3.2",
+    19: "1.3.3",
+    20: "1.3.4",
+    21: "1.3.5",
+    22: "1.3.6",
+    23: "1.3.7",
+    24: "1.3.8",
+    25: "1.4.1",
+    26: "1.4.2",
+    27: "1.4.3",
+    28: "1.4.4",
+    29: "1.4.5",
+    30: "1.4.6",
+    31: "1.4.7",
+    32: "1.4.8",
+    33: "1.5.1",
+    34: "1.5.2",
+    35: "1.5.3",
+    36: "1.5.4",
+    37: "1.5.5",
+    38: "1.5.6",
+    39: "1.5.7",
+    40: "1.5.8",
+    41: "1.6.1",
+    42: "1.6.2",
+    43: "1.6.3",
+    44: "1.6.4",
+    45: "1.6.5",
+    46: "1.6.6",
+    47: "1.6.7",
+    48: "1.6.8",
+}
+
+
+BASE_STATUS_OID = ".1.3.6.1.4.1.1718.3.2.3.1.10"
+BASE_CTRL_OID = ".1.3.6.1.4.1.1718.3.2.3.1.11"
+
+def _snmp_get(host, oid):
+    out = processwrapper.check_output(
```

Comment:
W already support `pysnmp` for SNMP and also include `pysnmp-mib` in `snmp-requirements.txt`. Please switch to using pysnmp directly instead of using an external program, i.e.
```suggestion
    import pysnmp.hlapi
```

and so forth here and in `_snmp_set`.

Reply:
Hmm, we already have the calls to the snmpset tool in `apc.py`, so I wouldn't mind doing the same here. Especially when this is only temporary before moving to pdudaemon.

Reply:
I didn't check this, in this case its fine to not convert to the python module.